### PR TITLE
MTDSA-5297 Updated service error mapping

### PR DESCRIPTION
**Pull request by Taiwo-AB**

### CODE REVIEW

File: it/v1/endpoints/RetrieveChargeHistoryControllerISpec.scala [hunk 1]
```diff
@@ -74,6 +74,39 @@ class RetrieveChargeHistoryControllerISpec extends IntegrationBaseSpec {
           response.json shouldBe mtdResponse
           response.header("Content-Type") shouldBe Some("application/json")
         }
+
+      }
+      "return a 500 status code" when {
+        "des returns multiple errors" in new Test {
```
Comment (narendravijayarao) [inline it/v1/endpoints/RetrieveChargeHistoryControllerISpec.scala:80]: all good, can you change the description something like this **des returns errors that map to DownstreamError**

Comment (BetaDraconis) [reply it/v1/endpoints/RetrieveChargeHistoryControllerISpec.scala:80]: but this test is specifically for the scenario in which multiple errors are returned from DES

Comment (BetaDraconis) [reply it/v1/endpoints/RetrieveChargeHistoryControllerISpec.scala:80]: never mind, I thought this was on the other service. Ignore me.

Comment (Taiwo-AB) [reply it/v1/endpoints/RetrieveChargeHistoryControllerISpec.scala]: Done

**Pull request merged**